skll/learner.py

```diff
@@ -72,24 +81,42 @@
                         [{'learning_rate': [0.01, 0.1, 1.0, 10.0, 100.0]}],
                         AdaBoostRegressor:
                         [{'learning_rate': [0.01, 0.1, 1.0, 10.0, 100.0]}],
+                        BayesianRidge:
+                        [{'alpha_1': [1e-6, 1e-4, 1e-2, 1, 10],
+                          'alpha_2': [1e-6, 1e-4, 1e-2, 1, 10],
+                          'lambda_1': [1e-6, 1e-4, 1e-2, 1, 10],
+                          'lambda_2': [1e-6, 1e-4, 1e-2, 1, 10]}],
                         DecisionTreeClassifier:
                         [{'max_features': ["auto", None]}],
                         DecisionTreeRegressor:
                         [{'max_features': ["auto", None]}],
                         DummyClassifier:
                         [{}],
+                        DummyRegressor:
+                        [{}],
                         ElasticNet:
                         [{'alpha': [0.01, 0.1, 1.0, 10.0, 100.0]}],
                         GradientBoostingClassifier:
                         [{'max_depth': [1, 3, 5]}],
                         GradientBoostingRegressor:
                         [{'max_depth': [1, 3, 5]}],
+                        HuberRegressor:
+                        [{'epsilon': [1.05, 1.35, 1.5, 2.0, 2.5, 5.0],
+                          'alpha': [1e-4, 1e-3, 1e-3, 1e-1, 1, 10, 100, 1000]}],
                         KNeighborsClassifier:
                         [{'n_neighbors': [1, 5, 10, 100],
                           'weights': ['uniform', 'distance']}],
                         KNeighborsRegressor:
                         [{'n_neighbors': [1, 5, 10, 100],
                           'weights': ['uniform', 'distance']}],
+                        MLPClassifier:
+                        [{}],
```

Comment:
I know searching over `hidden_layer_sizes` takes too long, but what about the other parameters like `activation`, `solver`, `learning_rate`, etc?